android/buildAndroidBOINC-CI.sh

```diff
@@ -239,54 +205,52 @@ getTripletName()
     echo $triplet_name
 }
 
-if [ $build_with_vcpkg = "yes" ]; then
-    if [ $ci = "yes" ]; then
-        triplets_setup="ci"
-    else
-        triplets_setup="default"
-    fi
-    manifest_dir=$THIRD_PARTY/vcpkg_ports/configs/$component
-    if [ $component = "apps" ]; then
-        manifest_dir=$manifest_dir/android
-    fi
-    manifests="--x-manifest-root=$manifest_dir --x-install-root=$VCPKG_ROOT/installed/"
-    vcpkg_overlay="--overlay-ports=$vcpkg_ports_dir/ports --overlay-triplets=$vcpkg_ports_dir/triplets/$triplets_setup"
-    vcpkg_flags="$vcpkg_overlay  --feature-flags=versions --clean-after-build"
-    if [ ! -d "$VCPKG_ROOT" ]; then
-        mkdir -p $BUILD_DIR
-        git -C $BUILD_DIR clone $VCPKG_LINK
-    fi
-    if [ ! -e /tmp/vcpkg_updated ]; then
-        git -C $VCPKG_ROOT pull
-        $VCPKG_ROOT/bootstrap-vcpkg.sh
-        touch /tmp/vcpkg_updated
-    fi
-    if [ $arch = "armv6" ]; then
-        export ANDROID_NDK_HOME=$NDK_ARMV6_ROOT
+if [ $ci = "yes" ]; then
+    triplets_setup="ci"
+else
+    triplets_setup="default"
+fi
+manifest_dir=$THIRD_PARTY/vcpkg_ports/configs/$component
+if [ $component = "apps" ]; then
+    manifest_dir=$manifest_dir/android
+fi
+manifests="--x-manifest-root=$manifest_dir --x-install-root=$VCPKG_ROOT/installed/"
+vcpkg_overlay="--overlay-ports=$vcpkg_ports_dir/ports --overlay-triplets=$vcpkg_ports_dir/triplets/$triplets_setup"
+vcpkg_flags="$vcpkg_overlay  --feature-flags=versions --clean-after-build"
+if [ ! -d "$VCPKG_ROOT" ]; then
+    mkdir -p $BUILD_DIR
+    git -C $BUILD_DIR clone $VCPKG_LINK
+fi
+if [ ! -e /tmp/vcpkg_updated ]; then
+    git -C $VCPKG_ROOT pull
+    $VCPKG_ROOT/bootstrap-vcpkg.sh
+    touch /tmp/vcpkg_updated
+fi
+if [ $arch = "armv6" ]; then
```

Comment:
In this block of code, only `armv6` (because of the NDK folder) and `arm` (because of the additional `neon` configuration) are different. Others could be safely combined in one statement

Reply:
It should remain for clearance of the script. Script also should be readable and not supper efficient.

Reply:
Less code duplication is still better, and makes the script clearer